cmd/autobump/config_test.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/go-faker/faker/v4"
+)
+
+func TestValidateGlobalConfig_Success(t *testing.T) {
+	// Arrange
```

Comment:
Could we enable "t.Parallel()" here? To run everything in parallel? I think it'll be very helpful though using Testify.